src/util.js

```diff
@@ -6,23 +6,14 @@
 plugin.service('wgnUtil', [
 	function() {
 
-		/**
-		 * Service
-		 *
-		 * @type {Object}
-		 */
-		var Service = this;
-
 		/**
 		 * Get a UUID
 		 *
 		 * @return {String} UUID
 		 */
 		this.getUUID = function() {
-			return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-				var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
-				return v.toString(16);
-			});
+		 return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g,
+		   function(c) { return parseInt(Math.random() * 16).toString(16); });
```

Comment:
@JoeBello that's an interesting change you made there, can you plz just explain the motivation?

Reply:
Linting failed due to use of the bitwise operators and the double equal.